docker/.env

```diff
@@ -0,0 +1,6 @@
+PATH_OMS_EVENTS=./
+BASE_URL=localhost.test
```

Comment:
does it mean that we'll need to use `http://localhost.test` instead of `http://localhost` on dev machines?

Reply:
the idea is that it can be set as anything one wants, so not really "need to"

of course it would be a good practice to use something that looks more like a real domain name, and the '.test' is a reserved domain from IANA, so it wont break stuff- one could use a nice tool like dnsmasq which works as local dns, see [here](https://blogging.dragon.org.uk/howto-setup-dnsmasq-as-dns-dhcp/)